### PR TITLE
Doc/update support email and do an overall edit of the README

### PR DESCRIPTION
**Pull request by PipKat**
Update the support email address in the README and do an overall edit. (The support email address in the project's TOML file was already updated.)

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -77,20 +79,20 @@ PyAEDT is part of the larger `PyAnsys <https://docs.pyansys.com>`_
...
-Ansys Maxwell, Ansys Q3D Extractor, Ansys Siwave, and Ansys Icepak using electrical CAD (ECAD) and
+`AEDT <https://www.ansys.com/products/electronics>`_ is a platform that enables true
+electronics system design. AEDT provides access to the Ansys gold-standard
+electro-magnetics simulation solutions, such as Ansys HFSS, Ansys Maxwell,
+Ansys Q3D Extractor, Ansys Siwave, and Ansys Icepak using electrical CAD (ECAD) and
 Mechanical CAD (MCAD) workflows.
-In addition, it includes direct links to the complete Ansys portfolio of thermal, fluid,
+
+In addition, AEDT includes direct links to the complete Ansys portfolio of thermal, fluid,
 and Mechanical solvers for comprehensive multiphysics analysis.
```
Comment (MaxJPRey) [inline README.rst:96]:
```suggestion
and mechanical solvers for comprehensive multiphysics analysis.
```

**Review by MaxJPRey — APPROVED** [review]
LGTM.

File: README.rst [hunk 2]
```diff
@@ -39,36 +39,38 @@ PyAEDT
...
 
     pip install pyaedt
 
-Install PyAEDT with all extra packages (matplotlib, numpy, pandas, pyvista):
+To install PyAEDT with all extra packages (Matplotlib, NumPy, Pandas, and PyVista),
+use this command:
 
 .. code:: python
 
     pip install pyaedt[full]
```
Comment (cutelolly) [inline README.rst:65]: or pip install pyaedt[all]

**Review by cutelolly — APPROVED** [review]

File: README.rst (hunk 2 above)
Comment (PipKat) [reply README.rst:65]: @cutelolly I had already merged this branch, but I think knowing this alternatively command is useful. Feel free to add it in a PR of your own!

**Pull request merged**